CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## 0.8.1 - 2018-06-06
+
 - Changed `btc::Network` (de)serializing into/from string
```

Comment:
Shouldn't we mark this change as `### Internal improvements` and add comment that we've moved to `rust-bitcoin 0.13.1`?

Reply:
Yes.